src/comb/template.clj

```diff
@@ -57,6 +57,6 @@
   ([source]
      (eval source {}))
   ([source bindings]
-     (let [keys (map (comp symbol name) (keys bindings))
+     (let [keys (apply vector (map (comp keyword name) (keys bindings)))
```

Comment:
Does it fail with 1.9.0-alpha14 then? If so, let's change this commit/PR to fix the `:keys` expansion, but we'll keep the version number as it is.